zerver/webhooks/honeybadger/doc.md

```diff
@@ -0,0 +1,27 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Honeybadger will use the `Honeybadger` stream by default if no stream is given
```

Comment:
We recommend lower case stream names, and the introductory section of this integration doc recommends the stream name "honeybadger". Can you change "Honeybadger" to "honeybadger" here and for the stream name in your first commit?

Reply:
sure